app/controllers/EightBallController.scala

```diff
@@ -0,0 +1,30 @@
+package controllers
+
+import javax.inject._
+
+import play.api.mvc._
+
+import models.EightBallReply
+
+@Singleton
+class EightBallController @Inject()(cc: ControllerComponents) extends AbstractController(cc) {
+
+  def randomReply = Action {
+    val rndString = EightBallReply.getRndBall.content
```

Comment:
while randomMessage it's more uniform and I can see why, I don't see the reason behind getRandomBall vs getRndBall